bucket/foobar2000-encoders.json

```diff
@@ -1,18 +1,36 @@
 {
     "version": "2020-07-11",
-    "homepage": "https://www.foobar2000.org/encoderpack",
     "description": "This pack includes every natively supported free encoder binary for use with the Converter foobar2000 component.",
+    "homepage": "https://www.foobar2000.org/encoderpack",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.foobar2000.org/license"
     },
     "url": "https://www.videohelp.com/download/Free_Encoder_Pack-2020-07-11.exe#/dl.7z",
     "hash": "87535c2b89da7915564c047bb6fa039345fdc7c7920c33c2882f789630df24e7",
-    "depends": "extras/foobar2000",
```

Comment:
Sorry for replying late, I've edited these manifests referring to the request, however, there's some questions before further edit:
1. As far as I known, it is symboliclink which needs admin rights, junction should work well with user rights, please let me know if I was wrong. 
2. Previously I deleted these two lines of notes because the manifest dose not create persist directory for foobar2000,
`"If you want to migrate persisted data to portable version,"`
`"please rename $persist_dir to \"foobar2000-portable\".`
please let me know if it should be kept anyway.
3. Importing scripts for foobar2000-portable rollback to previous version with minor changes, need further review.
Thanks a lot.